### PR TITLE
oo/213543 deploy application containerregistry

### PR DESCRIPTION
**Pull request by osagiestar**

### CODE REVIEW

File: infra/container-registry/application-container-registry.bicep [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+@description('Required. Environment name.')
+param environment string
+
+@description('Optional. Enable a single data endpoint per region for serving data. Not relevant in case of disabled public access. Note, requires the \'acrSku\' to be \'Premium\'.')
+param dataEndpointEnabled bool = false
+
+@description('Optional. Resource tags.')
+param tags object = {
+  Description: 'CDO Container Registry'
```
Comment (ppotturi) [inline infra/container-registry/application-container-registry.bicep:18]: @DaRoszko @kbabigumira do we need the Description tag as we already have the Purpose tag?

File: infra/container-registry/application-container-registry.bicep [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+@description('Required. Date in the format yyyyMMdd-HHmmss.')
+param deploymentDate string = utcNow('yyyyMMdd-HHmmss')
```
Comment (ppotturi) [inline infra/container-registry/application-container-registry.bicep:22]: Can you please change the description for deploymentDate and createdDate to Optional?

Comment (osagiestar) [reply infra/container-registry/application-container-registry.bicep:22]: Ok will look at it @ppotturi

Comment (osagiestar) [reply infra/container-registry/application-container-registry.bicep:22]: Done now @ppotturi

**Review by ppotturi — APPROVED** [review]

**Review by DaRoszko — APPROVED** [review]

**Review by ajaysoni-cap — APPROVED** [review]

**Pull request merged**